raft/src/test/java/org/apache/kafka/raft/MockLog.java

```diff
@@ -282,4 +282,11 @@ private EpochStartOffset(int epoch, long startOffset) {
             this.startOffset = startOffset;
         }
     }
+
+    public void close() {
```

Comment:
Maybe we could call this `clear` since it just resets the state and we're not overriding a method?